runtime/doc/autocmd.txt

```diff
@@ -787,6 +787,32 @@ ModeChanged			After changing the mode. The pattern is
 		:au ModeChanged [vV\x16]*:* let &l:rnu = mode() =~# '^[vV\x16]'
 		:au ModeChanged *:[vV\x16]* let &l:rnu = mode() =~# '^[vV\x16]'
 		:au WinEnter,WinLeave * let &l:rnu = mode() =~# '^[vV\x16]'
+Progress						*Progress*
+				After a progress-msg is message is created via
+				`nvim_echo`. The pattern is matched against
+				title of the message. The data item in autocmd
```

Comment:
matching against the title is unusual but makes sense I think

Reply:
I was thinking of matching against something like ID:TITLE . That might be more useful.